library/functools.po

```diff
@@ -101,19 +132,29 @@ msgid ""
 "*maxsize* and *currsize*.  In a multi-threaded environment, the hits and "
 "misses are approximate."
 msgstr ""
+"Pour aider à mesurer l'efficacité du cache et ajuster le paramètre "
+"*maxsize*, la fonction englobée est is instrumented avec une fonction :func:"
```

Comment:
*is instrumented* surveillée / analysée ? Voir un dictionnaire de synonyme à partir de surveillée doit y'avoir des bonnes pistes. Sinon surveillée c'est pas mal.